.pylintrc

```diff
@@ -0,0 +1,22 @@
+# Config file for pylint for CleverHans
+# Used by cleverhans.devtools
+
+[FORMAT]
+# TensorFlow 2 space style
+indent-string='  '
+
+# pylint should print out the file path, line number, and ID code for the
+# messages it prints. This is partly so we can turn off the ID codes that
+# we do not actually want to enforce.
+msg-template='{path}:{line}: [{msg_id}({symbol}), {obj}] {msg}'
+
+# ID codes to disable
+# C0103: Complains about the variable name 'x'
+# C0411: Import order
+# C0111: Missing docstring
+# W0613: Unused argument
```

Comment:
Are there any arguments not used in the API right now? Or is the attacks interface flagged as having unused arguments because of how we structured it around `parse_params` right now?

Reply:
I'm not 100% sure, but I think this warning is only triggered by the tutorials or examples.
I do think W0613 is one of the warnings we should prioritize turning back on though. Hopefully during the hackathon tomorrow.

Reply:
Just saw #597 so it's ok to discard for now